package.json

```diff
@@ -16,6 +16,7 @@
     "@types/react": "^17.0.30",
     "@types/react-dom": "^17.0.9",
     "axios": "^0.21.1",
+    "html-parse-stringify": "^3.0.1",
```

Comment:
Is this package used anywhere? Couldn't find a reference to it. Should remove if not using.